PsychSourceGL/Source/Common/Base/PythonGlue/PsychScriptingGluePython.c

```diff
@@ -462,16 +445,13 @@ PyObject* mxCreateString(const char* instring)
     #endif
 
     if (!ret) {
-        #if PY_MAJOR_VERSION < 3
-        // Fallback to standard C string decoding:
-        ret = PyString_FromString(instring);
-        #else
         // Try decoding assuming current system locale setting:
         ret = PyUnicode_DecodeLocale(instring, "surrogateescape");
         PyErr_Clear();
 
         // Retry with strict error handler, because of backwards incompatible
         // change in Python 3.6 -> 3.7 (sigh):
+        #ifndef Py_LIMITED_API
```

Comment:
A comment wouldn't hurt here, noting why this is ok. I guess the logic is that limited api is only used on Python 3.7+ and that on 3.7+ those fallback routines are no longer needed anyway?

Reply:
Exactly, but I'll add a comment stating that explicitly.